### ISSUE TITLE
bug: Android 5.0 Cannot read property 'getBoundingClientRect' of undefined

### ISSUE THREAD

**Issue by Jinoooo** [driftyco/ionic#4269]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>5.0</span>  <span ionic-webview>webview</span>

<span ionic-description>I am getting a lot of errors when bringing the keyboard up by focusing on the search field.
* *340677   error    Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined, http://192.168.0.15:8100/lib/ionic/js/ionic.bundle.js, Line: 3995*
* ionic 1.6.4
* keyboard plugin 1.0.4
* android 5.0
* device galaxy 5

Here is my code:
```
	<form ng-submit="search(data)">
		<ion-header-bar class="bar bar-subheader item-input-inset">
			<div class="item-input-wrapper">
				<i class="icon ion-ios7-search placeholder-icon"></i>
				<input type="search" placeholder="enter lastname ..." ng-model='data' autocorrect="off">
				<input type="submit" style="position: absolute; left: -9999px; width: 1px; height: 1px;" />
				<i class="clear-search icon ion-close-circled" ng-click="clearSearch()"></i>
			</div>
			<div class="padding item-input">
				<select ng-model="selected" ng-options="year for year in years"></select>
			</div>
		</ion-header-bar>
	</form>
```
</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @Jinoooo!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=4269). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=4269)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by mhartington** [comment]
Hmm, I'm not able to replicate this on my end. Could you provide a minimal demo (codepen, plunkr) that I can test against?

**Comment by Jinoooo** [comment]
I created a demo to simulate the problem, but no luck.  The main difference was the ionic version under lib/ionic.  I updated my app to the latest nighty version of ionic and now the error disappeared:
  "version": [-"1.0.0-rc.3",-]{+"1.1.0-nightly-1520",+}
Please close the bug.

**Comment by Jinoooo** [comment]
Unfortunately I had to switch back to stable version 1.0.0 and the problem is back.  I am sure the problem has been fixed in 1.1.0.  Also wasn't able to re-produce the problem for demo :(

**Comment by Ionitron** [comment]
Greetings @Jinoooo!

I've closed this issue because my sensors indicated it was old and inactive, and may have already been fixed in recent versions of Ionic. However, if you are still experiencing this issue, please feel free to reopen this issue by creating a new one, and include any examples and other necessary information, so that we can look into it further.

Thank you for allowing me to assist you.
